addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4328,7 +4328,7 @@ export class OdooEditor extends EventTarget {
             node.replaceChildren();
         }
 
-        sanitize(element, this.editable);
+        sanitize(element, element);
```

Comment:
`element` is the default for the second argument so you could just remove it here. Do you know why we introduced that distinction between the node to sanitize and the root? It's not there in 15 so I might have introduced that with the refactor but I don't remember why. Seeing the way it's used in 17 at least, I don't see an obvious reason why they shouldn't always be the same. And if there _is_ a good reason then this change here could be problematic.

Reply:
The commit that introduce `this.editable` as the root is the same as the commit that adds the root argument. (https://github.com/odoo/odoo/commit/55837e14d70e8161ef41de6ca723500d37b8c55e)

As there is no explanation that explains the use of `this.editable` in the second argument, we could assume that it was in case we called `OdooEditor.sanitize` on a node that was inside the editable.

As I don't see the use of the method sanitize on any node inside the editable but only an editable node, I believe it was not needed in the first place.

I've removed the second argument.